fasstq_filtrator/fastq_filtrator.py

```diff
@@ -0,0 +1,67 @@
+import argparse
+
+def filtrator (args):
+     with open(f"{args['output_file_prefix']}_passed.fastq", "w") as passed_read:
+        if args['save_filtered']:  
+            with open(f"{args['output_file_prefix']}_failed.fastq", "w") as failed_read:
+                with open(args['input_fastq'], "r") as fastq_file:
+		#reading fastq string-by-string 
+                    while True:
+                        seq_name = fastq_file.readline()
+                        seq = fastq_file.readline()
+                        seq_length = len(seq[:-1])
```

Comment:
Перенос строки лучше убирать при помощи `strip`. 
```suggestion
                        seq_length = len(seq.strip())
```
Почему нельзя слайс? В самой последней строке может не быть переноса (она же последняя), а значит ты отрежешь значимый символ и неправильно посчитаешь качество